components/style/gecko/pseudo_element.rs

```diff
@@ -75,10 +75,22 @@ impl PseudoElement {
         EAGER_PSEUDOS[i].clone()
     }
 
-    /// Whether this pseudo-element is ::before or ::after.
+    /// Whether the current pseudo element is :before or :after.
```

Comment:
Nit: two colons is the canonical spelling of these two pseudo-elements (even though due to CSS 2 a single colon is also allowed).  (Also I would still use a hyphen in "pseudo-element". :))